<ui version="4.0" >
 <class>FramesWindow</class>
 <widget class="QMainWindow" name="FramesWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Past Frames List</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView_" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionViewZoomIn_" />
    <addaction name="actionViewZoomOut_" />
    <addaction name="actionViewTogglePhantomCursor_" />
    <addaction name="separator" />
    <addaction name="actionShowMainWindow_" />
    <addaction name="actionShowPlayerWindow_" />
    <addaction name="separator" />
    <addaction name="actionPresetSave_" />
    <addaction name="actionPresetRevert_" />
   </widget>
   <addaction name="menuView_" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionViewZoomIn_" >
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="shortcut" >
    <string>I</string>
   </property>
  </action>
  <action name="actionViewZoomOut_" >
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="shortcut" >
    <string>O</string>
   </property>
  </action>
  <action name="actionViewTogglePhantomCursor_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Phantom Cursor</string>
   </property>
   <property name="shortcut" >
    <string>C</string>
   </property>
  </action>
  <action name="actionPresetSave_" >
   <property name="text" >
    <string>Save Preset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPresetRevert_" >
   <property name="text" >
    <string>Revert Preset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionShowMainWindow_" >
   <property name="text" >
    <string>Main Display</string>
   </property>
   <property name="shortcut" >
    <string>F2</string>
   </property>
  </action>
  <action name="actionShowPlayerWindow_" >
   <property name="text" >
    <string>Past Frames Player</string>
   </property>
   <property name="shortcut" >
    <string>F4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
